python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -56,14 +56,15 @@ def run_model(repo_id, test_api, in_out_pairs, local_model_hub=None, warm_up=1,
         result = run_ipex_fp16_gpu(repo_id, local_model_hub, in_out_pairs, warm_up, num_trials, num_beams)
 
     for in_out_pair in in_out_pairs:
-        results.append([repo_id,
-                        np.mean(result[in_out_pair], axis=0)[0],
-                        np.mean(result[in_out_pair], axis=0)[1],
-                        np.mean(result[in_out_pair], axis=0)[2],
-                        in_out_pair,
-                        f'{int(np.mean(result[in_out_pair], axis=0)[3])}' +
-                        f'-{int(np.mean(result[in_out_pair], axis=0)[4])}',
-                        num_beams])
+        if result is not None:
```

Comment:
do we need to initialize result=[] somewhere?

Reply:
`result` is initialized [here](https://github.com/intel-analytics/BigDL/blob/main/python/llm/dev/benchmark/all-in-one/run.py#L38)

Reply:
this is results not result.
I mean we need to initialize result in case native_int4 don't return anything?

Reply:
That's correct, sorry for the misunderstanding.